internal/convert.go

```diff
@@ -94,6 +97,11 @@ type NameAndCols struct {
 	Cols map[string]string
 }
 
+type FkeyAndIdxs struct {
```

Comment:
Brief comment describing the struct, like how we do in nameandcols.

Reply:
Done.